### DISCUSSION TITLE
Icon request: Roman Numerals (1 to 10)

### DISCUSSION THREAD

**Discussion by bbowers-hawaii** [FortAwesome/Font-Awesome#21232] [Icon Requests]
### Use case

Roman Numerals from I to X

### Example image

[https://stock.adobe.com/images/set-of-roman-numerals-vector-icon-on-white-isolated-background/309671644](url)

[https://www.freepik.com/premium-vector/roman-numerals-set-vector-illustration-isolated-white-background_30279379.htm](url)

### Icon request checklist

- [X] The title starts with "Icon request: " and is followed by the requested icon name
- [X] This icon does not represent a brand
- [X] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [X] The request is for a concrete object, or I've included an example image
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [X] I have [understood how requests work](https://fontawesome.com/community/leaderboard/new#faqs)

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

This is a request for **at least** 220 new icons: 2 families, classic and sharp, multiplied by 5 styles (solid, regular, light, thin, duotone), multiplied by 2 variants (stand-alone, circular), multiplied by the number of "roman literals", which is at least 11.

The main problem with the request is that it is unclear how many "roman literals" there are.

In example, `IV` is listed but `VI` isn't. It would be possible to get `VI` by flipping `IV`, but that would not fix the issue for `VII` or `VIII`. And the same applies to `XII` or any other unlisted value

I suggest to use a custom implementation with existing letters:
- https://fontawesome.com/icons/c?f=classic&s=solid
- https://fontawesome.com/icons/d?f=classic&s=solid
- https://fontawesome.com/icons/i?f=classic&s=solid
- https://fontawesome.com/icons/l?f=classic&s=solid
- https://fontawesome.com/icons/m?f=classic&s=solid
- https://fontawesome.com/icons/v?f=classic&s=solid
- https://fontawesome.com/icons/x?f=classic&s=solid

Demo: https://jsfiddle.net/tagliala/8djt9nur/13/

Hope it helps

Closing here, but feel free to comment

**Comment by bbowers-hawaii** [comment]
oh it was just a suggestion, not an exhaustive list of all the numerals desired.
Was hoping it would have been welcomed with, "What a great idea! We agree this would be a nice addition to our awesome service."
"We'll get started on roman numerals 1 to 10 and see how it goes, thanks!"

Anyway, thanks for the reply and for offering those alternatives. I had looked at using the letters prior to submitting. They require more width when combining which can be less ideal.

I do appreciate the amount of time and effort that goes into development and totally understand other priorities.
Mahalo

**Comment by tagliala** [comment]
@bbowers-hawaii 

so can this be considered a request for the first ten roman numerals (from I to X)?

**Comment by bbowers-hawaii** [comment]
@tagliala Sure! it would be much appreciated
The serif  "I" 
https://fontawesome.com/icons/i?f=classic&s=solid
is really the problem when combining due to the spacing though I appreciated your Demo
https://jsfiddle.net/tagliala/8djt9nur/13/

I think I could get by if you have or are able to develop an "I" without the serifs.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
